chronos.nim

```diff
@@ -5,5 +5,10 @@
 #              Licensed under either of
 #  Apache License, version 2.0, (LICENSE-APACHEv2)
 #              MIT license (LICENSE-MIT)
+
+## Chronos `async`/`await` framework
```

Comment:
It looks not very correct phrase,
I thought it should look like `Nim `async`/`await` framework` no?

Reply:
81a6584a44c69c03c47117c848dbbe63fac79fb5